PWGEM/Tasks/writer-skimmer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "OutputDirector": {
+    "debugmode": true,
+    "resfile": "V0tables",
+    "resfilemode": "RECREATE",
+    "ntfmerge": 1,
+    "OutputDescriptors": [
+      {
+        "table": "AOD/COLLISION/0"
+      },
+      {
+        "table": "AOD/V0DATA/0"
+      },
+      {
+        "table": "AOD/V0DATAEXT/0"
+      },
+      {
+        "table": "AOD/V0TRACKS/0"
+      },
+      {
+        "table": "AOD/V0INFOTRUE/0"
+      }
+    ]
+  }
+}
```

Comment:
I don't think we are supposed to keep these config files here. Or at least we don't have yet a guideline on this. What do you think @jgrosseo  ?

Reply:
okay, will take it out